docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+
+    graph_predictor = GraphPredictor(
+        name="Graphical model for tastiness of blended margaritas",
+        description="",
+        predictors=predictors_list,
+        training_data=[data_source] # the data source we defined above
+    )
+    graph_predictor = project.predictors.register(graph_predictor)
+
+One representation of this graphical model is shown below.
+
+.. figure:: _static/formulations_graphical_model.png
+    :align: center
+
+    Graphical model to predict tastiness of a margarita
+
+Defining a Design Space
+-----------------------
+
+Now that we have a trained predictor, the next step to finding the *world's best margarita* is to define the search space.
+A :class:`~citrine.informatics.design_spaces.formulation_design_space.FormulationDesignSpace` defines the ingredients, the labels, and any constraints.
+In addition to the labels used in the predictor, we also create a label for "simple syrup A" and "simple syrup B."
+This allows us to specify that one and only one of the simple syrups should be used.
+
+Notice that the design space's formulation descriptor corresponds to the original, unflattened formulation.
+That's because we want to describe the margarita we are physically making in our kitchen.
+The predictor takes care of flattening it.
+
+.. code-block:: python
+
+    from citrine.informatics.design_spaces import FormulationDesignSpace
+    from citrine.informatics.constraints import IngredientCountConstraint, IngredientFractionConstraint
+
+    fds = FormulationDesignSpace(
+        name="margaritas formulation,
+        description="",
+        formulation_descriptor=formulation,
+        ingredients={"simple syrup A", "simple syrup B", "tequila", "ice", "triple sec", "lime juice"},
+        labels={
+            "acid": {"lime juice"},
+            "alcohol": {"tequila", "triple sec"},
+            "sweetener": {"triple sec"},
+            "simple syrup": {"simple syrup A", "simple syrup B"}
+        },
+        constraints={
+            IngredientCountConstraint(formulation_descriptor=formulation, min=3, max=5),
+            IngredientCountConstraint(formulation_descriptor=formulation, min=1, max=1, label="simple syrup"),
+            IngredientFractionConstraint(formulation_descriptor=formulation, ingredient="lime juice", min=0.15, max=0.30)
+        }
+    )
+
+But that's not all; we also have to define the ``blend time`` input.
+We do this by wrapping a :class:`~citrine.informatics.design_spaces.product_design_space.ProductDesignSpace` around the formulation design space and a ``blend time`` dimension.
+
+.. code-block:: python
+
+    from citrine.informatics.design_spaces import ProductDesignSpace
+
+    design_space = ProductDesignSpace(
+        name="margaritas design space",
+        description="",
+        subspaces=[fds],
+        dimensions=[ContinuousDimension(descriptor=blend_time, lower_bound=5.0, upper_bound=20.0)]
+    )
+    design_space = project.design_spaces.register(design_space)
```

Comment:
MAYBE - do we want to mention that we can also use enumerated dimensions here, even if we don't demonstrate it? A lot of experimental folks are interested in those as well.

Reply:
Using an enumerated design space would require an `IngredientsToSimpleMixturePredictor`, and that's not included in this write-up since it's not being brought to beta yet.